package.json

```diff
@@ -37,6 +37,8 @@
     "@babel/core": "^7.11.6",
     "@babel/plugin-transform-runtime": "^7.11.5",
     "@babel/preset-env": "^7.11.5",
+    "@protobufjs/base64": "^1.1.2",
```

Comment:
I would hope that this could be removed, but we'll see.

Reply:
minified build went from 239 KiB -> 248 KiB (I should get that bundlesize action installed...)

So not that big a deal if we need it, although cool if we dont :D

Reply:
This would only be used by the loader, not included in the bundle. The increase in bundle size is probably the actual wasm module binary + the decoder code that I inlined.

Reply:
oh wow, thats pretty obvious about it not being in the bundle 🤦  Looks like you got rid of the dep though, thats cool!

Surprised the module binary + decoder took up that much. I'll have to keep an eye on that as we write more assembly script